src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -83,6 +83,9 @@
   private final static String DURATION_SUFFIX = ".duration.micros";
   private final static String OPERATION_NAME_TAG = "operationName";
 
+  private final static String SPAN_KIND_TAG_KEY = "kind";
```

Comment:
this should be span.kind instead of kind

Reply:
see https://github.com/opentracing/specification/blob/master/semantic_conventions.md for more details.